From 1aa2b139af4771e822e96153ffbfe7e3e4b43231 Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Thu, 20 Aug 2026 02:29:49 +0000
Subject: [PATCH] Update elements/extensions/mesa/mesa-sources.yml to
 mesa-26.1.8-0

Updates elements/extensions/mesa/mesa-sources.yml to mesa-26.1.8-0 (0fadfea)
Previous version was mesa-26.1.7-0 (e8617e4)
---
 elements/extensions/mesa/mesa-sources.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/extensions/mesa/mesa-sources.yml b/elements/extensions/mesa/mesa-sources.yml
index 7a60ba4fb..952dbfe50 100644
--- a/elements/extensions/mesa/mesa-sources.yml
+++ b/elements/extensions/mesa/mesa-sources.yml
@@ -6,7 +6,7 @@ sources:
   - '*rc*'
   - '*branchpoint*'
   - mesa-*.*.[01]
-  ref: mesa-26.1.6-0-gffa422e53d59a4938b38abd5c3fc319555da31dd
+  ref: mesa-26.1.8-0-g0fadfea4f394211946f308458f614839ef253ee8
 # Rust dependencies:
 - kind: tar
   directory: subprojects/packagecache/syn-2.0.87
-- 
2.55.0

